File: 0867-transpose-matrix/0867-transpose-matrix.java
class Solution {
    public int[][] transpose(int[][] matrix) {
        int [][] output = new int[matrix[0].length][matrix.length];
        for(int i=0;i< matrix.length;i++)
            for(int j=0; j<matrix[i].length; j++)
                output[j][i] = matrix[i][j];
        return output;
    }
}

File: 0867-transpose-matrix/0867-transpose-matrix.py
class Solution:
    def transpose(self, matrix: List[List[int]]) -> List[List[int]]:
        output = [[0]*len(matrix) for x in range(len(matrix[0]))]
        for i in range(len(matrix)):
            for j in range(len(matrix[i])):
                output[j][i] = matrix[i][j]
        return output